Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN ARCH=$(uname -m) && \
     dnf -y install \
       https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm \
       https://rpm.manageiq.org/release/19-spassky/el9/noarch/manageiq-release-19.0-1.el9.noarch.rpm && \
+    dnf -y --disablerepo=ubi-9-baseos-rpms swap openssl-fips-provider openssl-libs && \
     dnf -y module enable ruby:3.1 && \
```

Comment:
I debated adding a `dnf update` here too.  That would pull in any available updates from packages in Stream9, EPEL and ManageIQ repos.  It would get us closer to the same RPM package list as the appliances.